### ISSUE TITLE
Caseous vernixlike desquamation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7081]
**Preferred term label:**
Caseous vernixlike desquamation

**Synonyms**
Greasy, thick vernix caseosa-like scale 

**Definition (free text, please give PubMed ID)**
Vernix caseosa is a physiological, viscous biofilm that is produced by desquamated fetal skin and sebaceous glands covering the fetus at the third trimester in-utero. The substance's gross morphology in post-partum is described by the etymology, as vernix means varnish, and caseosa means cheesy-like matter. This finding refers to an abnormally thick and greasy vernix caseosa-like scale present at birth.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ichthyosis prematurity syndrome | 608649
PMID:21856041   23/23

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term. (Add annotation)

**Comment by pnrobinson** [comment]
Desquamation of skin soon after birth HP:0007549 is used for only two diseases 
- [ ] OMIM:608013 Gaucher Disease, Perinatal Lethal
- [ ]  OMIM:242300 Ichthyosis, Congenital, Autosomal Recessive 1

**Comment by pnrobinson** [comment]
Not a synonym, adding new term
